Doc/library/fileinput.rst

```diff
@@ -224,5 +224,5 @@ The two following opening hooks are provided by this module:
       Added the optional *errors* parameter.
 
    .. deprecated:: 3.10
-      This function is deprecated since :func:`input` and :class:`FileInput`
+      This function is deprecated since :func:`~fileinput.input` and :class:`FileInput`
```

Comment:
I would not hide the module name here.
(I do not mind if we have `fileinput.input` and `FileInput` without module, the second one is not ambiguous.)

Reply:
```suggestion
      This function is deprecated since :func:`fileinput.input` and :class:`FileInput`
```